### ISSUE TITLE
MegaMenu aria-label

### ISSUE THREAD

**Issue by timkingbu** [bu-ist/responsive-framework#315]
**Describe the bug**
“The aria-label should be used to provide a text alternative to an element that has no visible text on the screen.”

Main bug lies in responsive-foundation, issue has been moved to https://github.com/bu-ist/responsive-foundation/issues/276

Reopening... the aria-label in framework needs to match the visible label.
